CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 - [#564](https://github.com/mobilityhouse/ocpp/issues/564) Add support For Python 3.11 and 3.12
 - [#583](https://github.com/mobilityhouse/ocpp/issues/583) OCPP v1.6/v2.0.1 deprecate dataclasses from calls and call results with the suffix 'Payload'
 
+## BREAKING ##
```

Comment:
These changes are not yet breaking, I suggest to change the title to "Deprecated".

Reply:
Satisfied with commit https://github.com/mobilityhouse/ocpp/pull/580/commits/acba23f4678c62d5b171f6eaf2d1ea5924449299